### PR TITLE
Update front matter of multiple rules to acknowledge source of used assets

### LINKED ISSUE #1093: Cite source of rabbit video (and other resources?)
Many video rules are using the video of a rabbit, seems to be taken from https://www.w3schools.com/html/html5_video.asp itsel/ff coming from https://peach.blender.org/

I think we should cite our source. Probably a global note at the start of the Examples section in the line of "The video used in these examples come from…"

We should also check other sources and give credit accordingly if needed.

### PR DESCRIPTION
**Pull request by carlosapaduarte**
Update the front matter of rules that included
- moon-speech.mp3
- rabbit video
- web accessibility perspective video

to acknowledge the sources

### Closes issue(s):

- closes #1093 

### Need for Call for Review:

This will not require a Call for Review

---

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Call for Review period. In case of disagreement, the longer period wins.

### CODE REVIEW

**Review by JenniferChadwick — APPROVED** [review]

File: _rules/aria-state-or-property-permitted-5c01ea.md [hunk 1]
```diff
@@ -26,6 +26,8 @@ acknowledgments:
     - Jean-Yves Moyen
   funding:
     - WAI-Tools
+  assets:
+    - JFK's We Choose the Moon speech excerpt is courtesy of NASA.
```
Comment (Jym77) [inline _rules/aria-state-or-property-permitted-5c01ea.md:30]:
```suggestion
    - JFK's "We Choose the Moon" speech excerpt is courtesy of NASA.
```
Or in italics? Not sure which applies best: https://www.scribbr.com/mla/titles/
(needs to be done on all of them)

**Review by Jym77 — DISMISSED** [review]
Typographic nerd nitpicking… 🤓

File: _rules/aria-state-or-property-permitted-5c01ea.md (hunk 1 above)
Comment (carlosapaduarte) [reply _rules/aria-state-or-property-permitted-5c01ea.md:30]: I'm going with the quotation marks... according to your source, that's what seems most appropriate for an excerpt

**Review by WilcoFiers — CHANGES_REQUESTED** [review]
Could you remove the .DS_Store file?

**Review by Jym77 — APPROVED** [review]

**Comment by Jym77** [comment]
There's still a `.DS_store` file in the PR 🙈 
(you can set up a global `.gitignore` file on your system)

**Review by WilcoFiers — APPROVED** [review]

**Comment by carlosapaduarte** [comment]
> There's still a `.DS_store` file in the PR 🙈

Yes, but I guess it was in the repo beforehand. This PR initially modified it and then deletes it, so it should go away.

> (you can set up a global `.gitignore` file on your system)
Done!

**Comment by carlosapaduarte** [comment]
Call for review not needed. Merging.

**Pull request merged**